.github/workflows/test-python.yml

```diff
@@ -27,21 +31,17 @@ jobs:
       - name: Install libbson
         run: |
           LIBBSON_INSTALL_DIR=$(pwd)/libbson ./build-libbson.sh
-        working-directory: ./bindings/python
       - name: Install Python dependencies
         run: |
-          python -m pip install -U pip wheel
-          python -m pip install -r requirements/test.txt
-          python -c 'import pyarrow as pa; pa.create_library_symlinks()'
-        working-directory: ./bindings/python
+          python -m pip install -U pip
       - name: Install pymongoarrow
         run: |
-          # Use pip install to pull in install_requires dependencies
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -e .
-          # Re-compile to see compiler/linker output in the logs
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python setup.py build_ext --inplace --force
-        working-directory: ./bindings/python
+          # Install the library
+          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -vvv -e ".[test]"
       - name: Run tests
         run: |
           LD_LIBRARY_PATH=$(pwd)/libbson/lib python -m unittest discover test -v
-        working-directory: ./bindings/python
+      - name: Check the manifest
+        run: |
+          pip install check-manifest
+          check-manifest -v
```

Comment:
What is check-manifest and why is it needed?

Reply:
It makes sure that you are intentionally handling all of the files in version control and as built so you don't accidentally add/subtract files.

Reply:
Ack.